google/cloud/aiplatform/v1beta1/schema/trainingjob/definition_v1beta1/types/export_evaluated_data_items_config.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```

Comment:
The location of this file looks strange to me - why should this be under trainingjob/definition?

Reply:
The proto is shared between several other training job definitions (AutoMLForecasting, AutoMLTabular).